hbase-endpoint/src/test/java/org/apache/hadoop/hbase/coprocessor/TestSecureExport.java

```diff
@@ -389,11 +389,9 @@ public void testVisibilityLabels() throws IOException, Throwable {
         try (Connection conn = ConnectionFactory.createConnection(UTIL.getConfiguration());
           Table table = conn.getTable(importHtd.getTableName());
           ResultScanner scanner = table.getScanner(scan)) {
-          int count = 0;
-          for (Result r : scanner) {
-            ++count;
-          }
-          assertEquals(rowCount, count);
+          MutableInt count = new MutableInt();
+          scanner.forEach((r) -> count.increment());
```

Comment:
Unnecessary parentheses :)

And we could just use Iterables.size.

Reply:
lol, ok

Reply:
There was another module where I did MutableInt + forEach to count, will change that there too.